### ISSUE TITLE
Proposing Rhea: annotated biochemical reaction database ingest into dipper

### ISSUE THREAD

**Issue by putmantime** [monarch-initiative/dipper#533]
http://www.rhea-db.org/
About: http://www.rhea-db.org/webservice
Bulkdownload: ftp://ftp.ebi.ac.uk/pub/databases/rhea/biopax/
Terms of service: http://www.rhea-db.org/licensedisclaimer

Rhea comes from a joint ISB and EMBL-EBI effort  and annotates biochemical reactions including enzymes, kinetic data and participants (linking to ChEBI).

It's licensed under CC BY 4.0 allowing any type of use as long as it's attributed. 

There is an [api](http://www.rhea-db.org/webservice) as well as dumps in multiple formats including biopax2, and I have already done some parsing of the Rhea biopax2 dump using paxtools [here](https://github.com/putmantime/RheaReactionParser/blob/master/BioPaxJPype.ipynb)

With this data in monarch, we can make the enzyme to reaction connection through European Commision (EC) numbers that are already imported as xrefs from Kegg.  
This allows querying for substrates and products of enzymatic reactions; especially useful in modeling microbiome interactions between host and microbes.

**Comment by cmungall** [comment]
do you plan to use GO here too? We have rhea xrefs and will be improving these by axiomatizing rhea

**Comment by putmantime** [comment]
@cmungall yes that's great, I didn't realize GO had Rhea xrefs

**Comment by cmungall** [comment]
yes, and @bgood is generating OWL with all of the reaction participants in it
